<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1289</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QScrollArea" name="settingsArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>288</width>
         <height>877</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>870</height>
        </size>
       </property>
       <widget class="QGroupBox" name="ScanBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>250</width>
          <height>291</height>
         </rect>
        </property>
        <property name="title">
         <string>Scan</string>
        </property>
        <widget class="QPushButton" name="scanButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>240</y>
           <width>110</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>Continuous</string>
         </property>
        </widget>
        <widget class="QLabel" name="scanAmplitudeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Amplitude [V]</string>
         </property>
        </widget>
        <widget class="QLabel" name="scanOffsetLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Offset [V]</string>
         </property>
        </widget>
        <widget class="QLabel" name="scanFrequencyLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Frequency [Hz]</string>
         </property>
        </widget>
        <widget class="QLabel" name="scanWaveformLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>131</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>Shape</string>
         </property>
        </widget>
        <widget class="QComboBox" name="scanWaveform">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>150</y>
           <width>110</width>
           <height>34</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>34</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>Sine</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Square</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Triangle</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ramp Up</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ramp Down</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DC Voltage</string>
          </property>
         </item>
        </widget>
        <widget class="QDoubleSpinBox" name="scanAmplitude">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>28</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="scanOffset">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>68</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimum">
          <double>-2.000000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="scanFrequency">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>108</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
        <widget class="QPushButton" name="scanButtonManual">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>240</y>
           <width>110</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>Manual</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="scanSteps">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>193</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>10.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
         <property name="value">
          <double>100.000000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="scanStepsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>195</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Steps</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="LockingBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>670</y>
          <width>250</width>
          <height>200</height>
         </rect>
        </property>
        <property name="title">
         <string>Locking</string>
        </property>
        <widget class="QLabel" name="derivativeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
        <widget class="QLabel" name="proportionalLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>P</string>
         </property>
        </widget>
        <widget class="QLabel" name="integralLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>I</string>
         </property>
        </widget>
        <widget class="QPushButton" name="lockButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>150</y>
           <width>112</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="proportionalTerm">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>28</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="integralTerm">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>68</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="derivativeTerm">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>108</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="AcquisitionBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>250</width>
          <height>350</height>
         </rect>
        </property>
        <property name="title">
         <string>Acquisition</string>
        </property>
        <widget class="QPushButton" name="acquisitionButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>305</y>
           <width>110</width>
           <height>34</height>
          </rect>
         </property>
         <property name="text">
          <string>Acquire</string>
         </property>
        </widget>
        <widget class="QLabel" name="sampleRateLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Sample rate</string>
         </property>
        </widget>
        <widget class="QLabel" name="sampleNumberLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>131</width>
           <height>25</height>
          </rect>
         </property>
         <property name="text">
          <string>Sample number</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="chA">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>231</width>
           <height>95</height>
          </rect>
         </property>
         <property name="title">
          <string>Detector signal</string>
         </property>
         <widget class="QLabel" name="chARangeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>131</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Voltage range</string>
          </property>
         </widget>
         <widget class="QLabel" name="chACouplingLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>131</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Coupling</string>
          </property>
         </widget>
         <widget class="QComboBox" name="chACoupling">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>60</y>
            <width>90</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>AC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DC</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="chARange">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>110</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±50 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±100 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±200 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±500 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±1 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±2 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±5 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±10 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±20 V</string>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QGroupBox" name="chB">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>200</y>
           <width>231</width>
           <height>95</height>
          </rect>
         </property>
         <property name="title">
          <string>Reference signal</string>
         </property>
         <widget class="QLabel" name="chBRangeLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>131</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Voltage range</string>
          </property>
         </widget>
         <widget class="QLabel" name="chBCouplingLabel">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>131</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Coupling</string>
          </property>
         </widget>
         <widget class="QComboBox" name="chBCoupling">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>60</y>
            <width>90</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>AC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DC</string>
           </property>
          </item>
         </widget>
         <widget class="QComboBox" name="chBRange">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>20</y>
            <width>110</width>
            <height>25</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>Off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±50 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±100 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±200 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±500 mV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±1 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±2 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±5 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±10 V</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>±20 V</string>
           </property>
          </item>
         </widget>
        </widget>
        <widget class="QDoubleSpinBox" name="sampleRate">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>28</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="sampleNumber">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>68</y>
           <width>90</width>
           <height>30</height>
          </rect>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="singleStep">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="plottingArea" native="true">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>600</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>13</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="selectDisplay">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Detector signal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Reference signal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Error signal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Piezo voltage</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>677</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>34</height>
             </size>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QtCharts::QChartView" name="plotAxes" native="true">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1289</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header location="global">QtCharts/QChartView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
